packages/runtime/container-runtime/src/test/dataStoreContext.spec.ts

```diff
@@ -102,7 +102,9 @@ describe("Data Store Context Tests", () => {
                     createSummarizerNodeFn,
                     attachCb,
                     undefined,
-                    true /* isRootDataStore */);
+                    true /* isRootDataStore */,
```

Comment:
is it too much to just make these params as object, rather than spread like this? it will also give clearer naming with the property names

Reply:
could also do in a quick follow up if it is too much to bundle here

Reply:
I can do that as a follow up